core/src/com/mygdx/crazybomber/Player.java

```diff
@@ -0,0 +1,146 @@
+package com.mygdx.crazybomber;
+
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.g2d.Batch;
+import com.badlogic.gdx.graphics.g2d.Sprite;
+import com.mygdx.crazybomber.model.bomb.Bomb;
+
+import java.util.Stack;
+import java.util.concurrent.Executors;
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+public class Player extends Sprite {
```

Comment:
use this class to replace the player class we already have.

Reply:
ai